src/Polly.Extensions.Tests/Issues/IssuesTests.OnCircuitBreakWithServiceProvider_796.cs

```diff
@@ -0,0 +1,70 @@
+using Microsoft.Extensions.DependencyInjection;
+using Polly.CircuitBreaker;
+using Polly.Extensions.DependencyInjection;
+using Polly.Registry;
+
+namespace Polly.Core.Tests.Issues;
+
+public class IssuesTests
```

Comment:
Shall we make this partial too so we don't need to add it later?

Reply:
I tried but it complains about https://rules.sonarsource.com/csharp/RSPEC-2333.

Should be no problem to just add it later when we add more issues.

Reply:
@martincostello I just tried to add empty `IssuesTests` class as in other cases. It worked so it's non issue anyway.